### DISCUSSION TITLE
When We will get Solid Start?

### DISCUSSION THREAD

**Discussion by ghost** [solidjs/solid#817] [Q&A]
I am building a site with solid.js, fauna db, graphql & can't get a good performance score because that I am making the request for data (to faunadb using graphql) in the client & showing a loading spinner! but client side rendering slows down my app & performance score!

I like to use an ssr framework like next.js but i don't need a 300kbs bloat of js with out writing a single line of code!

I Really like to use html streaming which is experimental in next.js!

How can I fix my problems i see that solid start is in early days! What would you suggest me to do?

**Comment by orenelbaum** [comment]
You can use Solid Start already, it's usable but pretty unstable and like you said it's still in development, not even alpha. Alternatively you can use a framework agnostic SSR solution like vite-plugin-ssr or a static site generator like Astro if it fits your needs, or you can even handle SSR yourself using the SSR capabilities built into Solid.

**Reply by ghost** [reply]
Yes! I have tried to port my client side rendered solid.js app in to solid start & It works really well! but can't make a build because it says (solid-start build process) there is no entry point & also I need to deploy it to cloudflare pages?

![Screenshot (259)](https://user-images.githubusercontent.com/96905008/151749148-cdda8603-79d9-40bf-9b2f-26c469e5be79.png)

**Reply by ghost** [reply]
you said that I can handle SSR using the SSR capabilities built into Solid. but how?

**Reply by maxmilton** [reply]
@ZeAmanuel28 check out <https://www.solidjs.com/guides/server>, <https://github.com/solidjs/solid/tree/main/packages/solid-ssr>, and <https://github.com/search?q=org%3Asolidjs+ssr&type=code>.

**Reply by ghost** [reply]
Thanks for helping me out!
